src/transformers/pipelines/__init__.py

```diff
@@ -187,9 +190,8 @@
         "tf": (TFAutoModelForTableQuestionAnswering,) if is_tf_available() else (),
         "default": {
             "model": {
-                "pt": "google/tapas-base-finetuned-wtq",
-                "tokenizer": "google/tapas-base-finetuned-wtq",
```

Comment:
deleted tokenizer because of https://github.com/huggingface/transformers/pull/17667/files#r910987282